packages/cli/src/utils/changelog.js

```diff
@@ -14,7 +51,7 @@ const getChangelogFromMarkdown = (version, markdown) => {
   );
 
   if (startingLine === -1) {
-    return '';
+    throw Error('Invalid start position');
```

Comment:
This error gets swallowed [here](https://github.com/zapier/zapier-platform/pull/653/files#diff-a926e0ee55f23ed85c0f676cb5556f3e4d507ffc05775c8f441725d9f425bba8R101)

Reply:
Yes; previously it returned early with an empty string; given we're using a Promise to handle this I'm reusing the `catch` handler as a general error flow instead of repeating the default value.